data.sql

```diff
@@ -1,5 +1,7 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
```

Comment:
- [ ] There's a spelling error in the field `escap_attempts`. The animals table carries `escape_attemps` but you're referencing to `escape_attempts` while populating data into that field. Kindly check and fix it.

### Error
![errro-2](https://github.com/theresetuyi/Vet-clinic-database/assets/26597432/3e1bf137-1e76-4882-93aa-48f7a368bb90)